src/esp/sim/Simulator.cpp

```diff
@@ -1339,5 +1343,32 @@ int Simulator::getAgentObservationSpaces(
   }
   return spaces.size();
 }
+
+std::vector<std::string> Simulator::getRuntimePerfStatNames() {
+  return {"num rigid",           "num active rigid",    "num artic",
+          "num active overlaps", "num active contacts", "draw count",
+          "draw faces"};
```

Comment:
Nit: I would use `drawable` instead of the shortened `draw` to ensure that this doesn't get misinterpreted.